library/src/main/java/com/alexstyl/contactstore/ContactQueries.kt

```diff
@@ -233,10 +233,11 @@ internal class ContactQueries(
                         nickname = row[NicknameColumns.NAME]
                     }
                     GroupColumns.CONTENT_ITEM_TYPE -> {
-                        val groupId = row[GroupColumns.GROUP_ROW_ID].toLong()
-                        val id = row[GroupColumns._ID].toLong()
-                        groupIds.add(GroupMembership(_id = id, groupId = groupId))
-                        Unit
+                        val groupId = row[GroupColumns.GROUP_ROW_ID].toLongOrNull()
+                        val id = row[GroupColumns._ID].toLongOrNull()
```

Comment:
AFAIK all rows in the provider have an ID (do correct me if I am wrong), so I think it's best to assume that it will always be there. Let's keep the id as is (use `.toLong()`). If we do not have an id and the user of ContactStore need to edit the value, they won't be able to do so.

Reply:
Assumption is the mother of all.. you probably know the saying. I would prefer to always safely parse a String to a Number and does something else when the parsing fails.